src/cljs/athens/parser.cljs

```diff
@@ -38,7 +38,10 @@
      :bref (fn [id]
              (let [string (subscribe [:block/string [:block/uid id]])]
                [:span {:style {:font-size "0.9em" :border-bottom "1px solid gray"}}
-                [:a {:href (rfee/href :page {:id id})} (parse (:block/string @string))]]))}
+                [:a {:href (rfee/href :page {:id id})} (parse (:block/string @string))]]))
+     :bold (fn [text]
+             [:span
+              [:strong text]])}
```

Comment:
I would return just `[:strong text]` – the extra `span` wrapper doesn't add anything in this case.